test/integration/shared/lockup-dynamic/LockupDynamic.t.sol

```diff
@@ -21,130 +21,130 @@ abstract contract LockupDynamic_Integration_Shared_Test is Lockup_Integration_Sh
+        _params.createWithDurations.recipient = users.recipient;
+        _params.createWithDurations.totalAmount = defaults.TOTAL_AMOUNT();
+        _params.createWithDurations.asset = dai;
+        _params.createWithDurations.cancelable = true;
+        _params.createWithDurations.transferable = true;
+        _params.createWithDurations.broker = defaults.broker();
+
+        _params.createWithTimestamps.sender = users.sender;
+        _params.createWithTimestamps.recipient = users.recipient;
+        _params.createWithTimestamps.totalAmount = defaults.TOTAL_AMOUNT();
+        _params.createWithTimestamps.asset = dai;
+        _params.createWithTimestamps.cancelable = true;
+        _params.createWithTimestamps.transferable = true;
+        _params.createWithTimestamps.startTime = defaults.START_TIME();
+        _params.createWithTimestamps.broker = defaults.broker();
 
         // See https://github.com/ethereum/solidity/issues/12783
         LockupDynamic.SegmentWithDelta[] memory segmentsWithDeltas = defaults.segmentsWithDeltas();
         LockupDynamic.Segment[] memory segments = defaults.segments();
         for (uint256 i = 0; i < defaults.SEGMENT_COUNT(); ++i) {
-            _params.createWithDeltas.segments.push(segmentsWithDeltas[i]);
-            _params.createWithMilestones.segments.push(segments[i]);
+            _params.createWithDurations.segments.push(segmentsWithDeltas[i]);
+            _params.createWithTimestamps.segments.push(segments[i]);
         }
     }
 
     /// @dev Creates the default stream.
     function createDefaultStream() internal override returns (uint256 streamId) {
-        streamId = lockupDynamic.createWithMilestones(_params.createWithMilestones);
+        streamId = lockupDynamic.createWithTimestamps(_params.createWithTimestamps);
     }
 
     /// @dev Creates the default stream with the provided asset.
     function createDefaultStreamWithAsset(IERC20 asset) internal override returns (uint256 streamId) {
-        LockupDynamic.CreateWithMilestones memory params = _params.createWithMilestones;
+        LockupDynamic.CreateWithTimestamps memory params = _params.createWithTimestamps;
         params.asset = asset;
-        streamId = lockupDynamic.createWithMilestones(params);
+        streamId = lockupDynamic.createWithTimestamps(params);
     }
 
     /// @dev Creates the default stream with the provided broker.
     function createDefaultStreamWithBroker(Broker memory broker) internal override returns (uint256 streamId) {
-        LockupDynamic.CreateWithMilestones memory params = _params.createWithMilestones;
+        LockupDynamic.CreateWithTimestamps memory params = _params.createWithTimestamps;
         params.broker = broker;
-        streamId = lockupDynamic.createWithMilestones(params);
+        streamId = lockupDynamic.createWithTimestamps(params);
     }
 
     /// @dev Creates the default stream with deltas.
     function createDefaultStreamWithDeltas() internal returns (uint256 streamId) {
```

Comment:
Wouldn't it sound better to be called function `createDefaultStreamWithDurations`? This function calls function `lockupDynamic.createWithDurations` with `_params.createWithDurations` so the implementation does not appear to be consistent with the function name.